packages/block-editor/src/components/list-view/block-select-button.js

```diff
@@ -194,7 +194,7 @@ function ListViewBlockSelectButton(
 							{ images.map( ( image, index ) => (
 								<span
 									className="block-editor-list-view-block-select-button__image"
-									key={ `img-${ image.url }` }
+									key={ index }
```

Comment:
Using the `index` as a `key` isn't recommended. Maybe we could use `clientId` here.

Reply:
Thanks for the review!

> Using the `index` as a `key` isn't recommended. 

I did not know about this 😅 I have taken the following actions:

- I have renamed the `getImageUrl` function to the `getImage` function. This is because this function returns an object containing information about the image.
- Include the `clientId` in the return value of the `getImage` function and use it as a key for the map function.